pkg/cloud/cloud.go

```diff
@@ -776,7 +781,17 @@ func (c *cloud) IsExistInstance(ctx context.Context, nodeID string) bool {
 	return true
 }
 
+var lastCreateSnapshot sync.Map
+
 func (c *cloud) CreateSnapshot(ctx context.Context, volumeID string, snapshotOptions *SnapshotOptions) (snapshot *Snapshot, err error) {
+	if lastTimeUncast, ok := lastCreateSnapshot.Load(volumeID); ok {
+		if lastTime, ok := lastTimeUncast.(time.Time); ok {
+			if time.Since(lastTime) < time.Second*30 {
```

Comment:
Should be 15s. https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html